packages/solana-adapter/src/getCancelPaymentSignedTx.ts

```diff
@@ -0,0 +1,68 @@
+import { Connection, SystemProgram, PublicKey } from '@solana/web3.js';
+import { getAssociatedTokenAddress, TOKEN_PROGRAM_ID } from '@solana/spl-token';
+import { Program } from '@project-serum/anchor';
+import { AnchorWallet } from '@solana/wallet-adapter-react';
+import { HelioIdl } from './program';
+import { CancelPaymentRequest } from './types';
+import { helioFeeWalletKey, daoFeeWalletKey } from './config';
+import { signTransaction } from './utils';
+
+export const getCancelPaymentSignedTx = async (
+  connection: Connection,
+  wallet: AnchorWallet,
+  program: Program<HelioIdl>,
+  req: CancelPaymentRequest
+): Promise<string> => {
+  const [pda] = await PublicKey.findProgramAddress(
+    [req.payment.toBytes()],
+    program.programId
+  );
+  const mint = req.mintAddress!;
+
+  const senderAssociatedTokenAddress = await getAssociatedTokenAddress(
```

Comment:
you can name this senderTokenAccount, and then pass to object only value, without key
{
 senderTokenAccount
}
the same about other 4

Reply:
Ok